src/Sulu/Component/Webspace/Exception/NoValidWebspaceException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace\Exception;
+
+/**
+ * Runtime exception if no valid webspaces are found
+ *
+ * @author Patrik Karisch <patrik@karisch.guru>
+ */
+class NoValidWebspaceException extends \RuntimeException
```

Comment:
The inheritors of `WebspaceException` are supposed to have errors in webspaces. But in this case there is no valid webspace..